apis/v1beta1/httproute_types.go

```diff
@@ -199,7 +199,8 @@ type HTTPRouteRule struct {
 	// URLRewrite and RequestRedirect filters, which may not be combined. If an
 	// implementation can not support other combinations of filters, they must clearly
 	// document that limitation. In all cases where incompatible or unsupported
-	// filters are specified, implementations MUST add a warning condition to status.
+	// filters are specified, implementations MUST add the `PartiallyInvalid` error
+	// condition with status `True` and reason `IncompatibleFilters`.
```

Comment:
What would implementations be expected to do with a partially invalid route? Generally we've had the high level idea of failing fast when we encounter a broken/invalid state. This seems to indicate that implementations would continue to implement the rest of the route without these filters?

Edit: On further reading it's clearer to me that this condition would be used when there was at least one valid and one invalid rule present in the route. It would be helpful to clarify that here and potentially elsewhere in the spec.

Reply:
I think this is covered, please reopen if not!